SI/Sem.02/Solutions/Task06.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+#include <cmath>
+using namespace std;
+
+int main()
+{
+	double coefA, coefB, coefC;
```

Comment:
coef is not necessary in this case in my opinion.

Also , try to have a more User-friendly input:
For example:
	 cout << "Enter the first number: ";
	 cin >> a;